### DISCUSSION TITLE
Palindrome Checker

### DISCUSSION THREAD

**Discussion by salahsaeed19** [community/community#65339] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Write a function that takes a string as input and returns whether the string is a palindrome (reads the same forwards and backwards).

**Comment by Mohammed-Aljazzar** [comment]
def is_palindrome(s):
    s = s.lower().replace(" ", "")
    return s == s[::-1]

# Test cases
print(is_palindrome("racecar"))  # True
print(is_palindrome("hello"))    # False
print(is_palindrome("A man a plan a canal Panama"))

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
